script/grblcon.py

```diff
@@ -159,19 +180,40 @@ class GrblCon:
         'break': '/0 Print a pagebreak',
         'clear': '/0 Clear the screen',
         'close': '/0 Close the current grbl connection',
+        'dump_eeprom': ''.join([
+            '/0 Tells GRBL to print out the contents ',
+            'of its eeprom settings'
+        ]),
         'help': '/0 Display this dialog',
-        'home': '/0..4 (X/Y/Z/C) Homes specified axis.'
-                'Multiple axes may be provided (or none to home all)',
+        'home': ''.join([
+            '/0..4 (X/Y/Z/C) Homes specified axis.',
+            'Multiple axes may be provided (or none to home all)'
```

Comment:
I don't have a strong preference, but the `join` and the list operator do seem unnecessary. I didn't realize that emacs balked at broken strings like that, though.